node/utils/products.ts

```diff
@@ -51,3 +51,9 @@ export const mapSkus = (skuURL: string) => {
     }, []) ?? []
   )
 }
+const decimalNumber = (num: number): string | number => {
+    if(num === undefined) return num
+    const dec = num.toString().split('.')[1]
+    const len = dec && dec.length > 2 ? dec.length : 2
+    return Number(num).toFixed(len)
```

Comment:
your method is getting a number and returning a string, you could improve it

Reply:
Yes, this is the solution. I explain: Javascript by default transform a number with decimals .00 to without decimal, example if one number is 10.00 javascript being a number, shows it without the decimals, by this motive the solution is return string.
IMPORTANT: I tested in the template of the mail and there was not problem at receive this data with a type string. I test with the helpers of handlerbars for check and not harm anything that is implemented.